boefjes/plugins/kat_webpage_analysis/main.py

```diff
@@ -2,26 +2,40 @@
 from typing import Tuple, Union, List
 from boefjes.job_models import BoefjeMeta
 
+from os import getenv
 import requests
+from requests import Session
+
 from urllib.parse import urlparse, urlunsplit
 from forcediphttpsadapter.adapters import ForcedIPHTTPSAdapter
 
 
+ALLOWED_CONTENT_TYPES = [
```

Comment:
We should use the python native 'mimetypes' module, and its mimetypes map for this as we can accept a wide range of mimetypes here.
Specifically, we would need to make sure to *not* accept anything that KAT would produce itself (eg, boefje, kat-* mimetypes), but since we don't know all of those yet, lets limit on recognized types instead.
```import mimetypes
ALLOWED_CONTENT_TYPES = mimetypes.types_map.values
```

Reply:
Fixed